skypilot-config.yaml

```diff
@@ -383,6 +383,10 @@
 workdir: .
 resources:
   accelerators: ["H100:1", "A100-80GB:1"]
+  ports:
+    - 8080
```

Comment:
Not sure if this port is necessary. It was exposed by default and I haven't tested without it.

Reply:
Removed, was not necessary.